packages/ckeditor5-remove-format/tests/integration.js

```diff
@@ -65,6 +66,19 @@ describe( 'RemoveFormat', () => {
 					'<imageBlock src="assets/sample.png"><caption>caption</caption></imageBlock><paragraph>bar]</paragraph>' );
 		} );
 
+		it( 'does not remove image width attribute', () => {
+			setModelData( model, '<paragraph>foo[<$text bold="true">foo</$text></paragraph>' +
+				'<imageBlock src="assets/sample.png" width="50%"><caption>caption</caption></imageBlock>' +
+				'<paragraph>bar]</paragraph>' );
```

Comment:
This unit test can be further simplified. The only goal of your test is to check whether width attribute is removed from a block image. You don't care about paragraphs.

So let's just remove them. Simplified your test can look like that:

```
	it( 'does not remove image width attribute', () => {
		setModelData( model, '[<imageBlock src="assets/sample.png" width="50%"><caption>caption</caption></imageBlock>]' );

		editor.execute( 'removeFormat' );

		expect( getModelData( model ) )
			.to.equal( '[<imageBlock src="assets/sample.png" width="50%"><caption>caption</caption></imageBlock>]' );
	} );
```

Note that after doing this the test no longer **has no longer multiline range**, while it is contained in a [group that tests multiline ranges](https://github.com/ckeditor/ckeditor5/blob/9557aea925b968cfd3ba82810b1ef09e96c03164/packages/ckeditor5-remove-format/tests/integration.js#L47). So you'll have to move it outside of this group (IMHO it make sense to put it directly in [RemoveFormat test suite](https://github.com/ckeditor/ckeditor5/blob/9557aea925b968cfd3ba82810b1ef09e96c03164/packages/ckeditor5-remove-format/tests/integration.js#L24).